dropwizard-logging/src/main/java/io/dropwizard/logging/AbstractAppenderFactory.java

```diff
@@ -149,6 +149,12 @@ public TimeZone getTimeZone() {
     }
 
     @JsonProperty
+    public void setTimeZone(String zoneId) {
+        this.timeZone = zoneId.equalsIgnoreCase("system") ? TimeZone.getDefault() :
```

Comment:
The method should be invoked in that case, but that a good idea in the spirit of defensive programming. I'm not a big fan of  reversed conditions (they don't look natural for me), so I will just use `Strings.nullOrEmpty`.